src/recordlinker/schemas/seed.py

```diff
@@ -0,0 +1,41 @@
+import uuid
+
+import pydantic
+
+from recordlinker.schemas.mpi import PatientRef
+from recordlinker.schemas.pii import PIIRecord
+
+CLUSTER_MAX_SIZE = 100
+# NOTE: the number of records in each cluster can vary, but assuming we have
+# less than ~25 records per person, we should be able to fit 100 persons in a
+# 1MB request.  This is a reasonable limit for a single request.
```

Comment:
Should there also be a `RECORDS_MAX_SIZE` to enforce the 25 record limit per cluster assumption?

Reply:
This is a tricky one.

The 1MB limit isn't a hard rule. Many web servers can handle, or be configured to handle, payloads much larger than 1MB.  However, most are configured with some limit. I used nginx as a baseline here, since it's a very popular web server to put in front of application servers, and its [default value is 1MB](https://nginx.org/en/docs/http/ngx_http_core_module.html#client_max_body_size).

This API endpoint will be able to process more than that.  I'm sure it could probably handle a 5MB payload with no problem (assuming there is enough available RAM to use).  However, if the user were to send a really large payload here, and the web server fronting this application had a smaller limit, the error would happen before RL was even called.  As nginx or Apache would raise an error when it receives a body that is too large, thus this check wouldn't even happen.

More than anything, this kind of acts as documentation / guidance, on how the user should consider using this endpoint.  The array length limits really won't enforce anything on size.  It's conceivable, albeit unlikely, for just a single PII document to pass the 1MB threshold, say if they had 10000 name values or 10000 addresses or something absurd like that.  Nonsense, of course, but possible.

It might make more sense for us to just put the threshold on body size, but my worry is that would be harder for the user to control when sending requests to this API.  It just much easier for them to use some sort of page size limit.

Reply:
Yeah you're right. Either way its not actually enforcing a size (like if the records are huge or there are a ton of them etc). I guess it makes sense to not add a ~25 patient limit to a cluster. I think the cluster limit you already have is a good idea if the goal is mainly guidance. I'm fine keeping it like this.

Reply:
We don't really have a body size limit on any other endpoint either so I'm good with how it is